package.json

```diff
@@ -12,8 +12,9 @@
   },
   "dependencies": {
     "@headlessui/react": "^1.7.17",
+    "axios": "^1.6.2",
     "eslint-plugin-tailwindcss": "^3.13.0",
-    "next": "13.5.5",
+    "next": "^14.0.4",
```

Comment:
Why next upgrade?

Reply:
Because I had issues with tailwind classes and phind suggested to update next

Reply:
I have downgrade it as it was